app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  validates :username, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true
+  validates :first_name, presence: true
+  validates :phone_number, presence: true
```

Comment:
Just out of curiosity, is it better to have each validation on a separate line like it is above, or to put similar traits on the same line, like so:

```ruby
validates :username, :email, presence: true, uniqueness: true
```

I've always done it the latter, but I think I like the way it's done here better. It's more readable.